DataFormats/Detectors/CTP/include/DataFormatsCTP/CTPRateFetcher.h

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef COMMON_CCDB_CTPRATEFETCHER_H_
+#define COMMON_CCDB_CTPRATEFETCHER_H_
+
+#include <string>
+
+#include "CCDB/BasicCCDBManager.h"
+#include "DataFormatsParameters/GRPLHCIFData.h"
+
+namespace o2
+{
+namespace ctp
+{
+class CTPRunScalers;
+class CTPConfiguration;
+
+class CTPRateFetcher
+{
+ public:
+  CTPRateFetcher() = default;
+  double fetch(o2::ccdb::BasicCCDBManager* ccdb, uint64_t timeStamp, int runNumber, std::string sourceName);
```

Comment:
since other changes are needed, please also change `std::string sourceName` to `const std::string& sourceName`, also below.